pkg/crcontrollers/reservation/reservationcontroller.go

```diff
@@ -135,3 +164,39 @@ func (c *Controller) handleReservationUpdate(ctx context.Context, log *logrus.En
 		return ctrl.Result{}, nil
 	}
 }
+
+func (c *Controller) setReservationParameters() {
+	var (
+		fastTimeoutStr = os.Getenv(fastTimeoutEnv)
+		slowTimeoutStr = os.Getenv(slowTimeoutEnv)
+		maxAttemptsStr = os.Getenv(attemptsNumEnv)
+
+		fastTimeout time.Duration
+		slowTimeout time.Duration
+		maxAttempts uint64
+		err         error
+	)
+
+	fastTimeout, err = time.ParseDuration(fastTimeoutStr)
```

Comment:
`0s` and `0` also parsable and won't raise an error. The below code too.

Reply:
It is valid argument in RateLimiter, which means no timeout